cli/integration/tests/waiter/util.py

```diff
@@ -110,6 +110,12 @@ def load_token_with_headers(waiter_url, token_name, assert_response=True, expect
     return response.json(), response.headers
 
 
+def log_curl(waiter_url, endpoint, headers, params, payload):
+    curl_header_flags = ' '.join(f"-H '{k}: {v}'" for k, v in headers.items())
+    curl_url = f'{waiter_url}{endpoint}?{"&".join((k + "=" + v) for k, v in params.items())}'
+    logging.debug(f'curl -XPOST {curl_header_flags} -d \'{json.dumps(payload)}\' \'{curl_url}\'')
```

Comment:
Nitpick: can we use double quotes for the string and avoid escaping the single quotes?